database/seeder/generator.js

```diff
@@ -70,6 +56,8 @@ const generateBiz = () => {
     const type = [];
     type.push(_.sample(typeOne));
```

Comment:
Wow! I guess the great thing about looking at someone else's code is to see cool tricks like .sample. Great implementation.